docs/examples/rx/03_listen_to_balance_change/index.js

```diff
@@ -0,0 +1,46 @@
+const { switchMap } = require('rxjs/operators');
+
+// Import the API Rx
+const { ApiRx } = require('@polkadot/api');
+
+// Known account we want to use (available on dev chain, with funds)
+const Alice = '5GoKvZWG5ZPYL1WUovuHW3zJBWBP5eT8CbqjdRY4Q6iMaDtZ';
+
+function main () {
+  let previous = undefined;
+
+  // Here we pass the (optional) provider
+  // We avoid having nested `.subscribe` blocks. Use `.subscribe` or
+  // `.tap` blocks for debugging with console.log
+  ApiRx
+    .create()
+    .pipe(
+      // Here we subscribe to any balance changes and update the on-screen value.
+      // Use the Storage chain state (runtime) Node Interface.
+      switchMap((api) => api.query.balances.freeBalance(Alice))
+    )
+    .subscribe((current) => {
+      previous = previous || current;
+
+      // FIXME - refactor like the ApiPromise where only the balance change console.log's
```

Comment:
Not convinced we actually want FIXMEs in the examples at all. It should just be simple and working.